### DISCUSSION TITLE
SFTP write error

### DISCUSSION THREAD

**Discussion by ppetersen123** [nextgenhealthcare/connect#5523] [Q&A]
Have an issue with an SFTP write error in Mirth 4.1.1.  The Destination Settings for the Channel are correct for SFTP, yet the transaction bounces back with sftp write connect error permission denied.  If that private IP address of 10.1.1.60 is configured nowhere in Mirth and the test write works what is happening?  Thanks.

**Comment by NoraTheExplorer** [comment]
Have you tried to connect with the same credentials via a FTP client and then write a file? It might be a permission problem.

Can you post the exact error message?

**Comment by pacmano1** [comment]
What does "If that private IP address of 10.1.1.60 is configured nowhere in Mirth" mean?

Also, while this is not the mirth forums, please read the forum etiquette at https://forums.mirthproject.io/articles/177082-forum-etiquette, much of which applies here when asking for help.

**Comment by pacmano1** [comment]
Moving to Discussions.
